logs_analyzer/chapter2/scala/src/main/scala/com/databricks/apps/logs/ApacheAccessLog.scala

```diff
@@ -0,0 +1,35 @@
+package com.databricks.apps.logs
+
+/**
+  * An entry of Apache access log.
+  */
+case class ApacheAccessLog(ipAddress: String,
```

Comment:
Only if reorganize the code. Currently all chapters are independent projects. The other option could be to group code by a programming language for all chapters. Anyway, not in this pull request.